Ada Examples/Examples Based On OpenGL Superbible/Simplest/src/shaders_program.ads

```diff
@@ -0,0 +1,11 @@
+
+with GL.Objects.Programs;
+
+with Window_Types;
+
+package Shaders_Program is
+
+    function Make_Shader_Program(Window         : Window_Types.tWindow;
+                                 Shader_Program : out GL.Objects.Programs.Program) return Boolean;
```

Comment:
I prefer to keep the shader programming code separate as it seems to me conceptually different to the rest of the code. Just my upbringing which orients me to not mixing apples and oranges I suppose, or weapons systems and sensor systems (reflecting back to my military software design experiences).
Possibly an over-reaction.
If you really prefer it elsewhere, no problem.